packages/react-components/lib/utils/item-table.ts

```diff
@@ -0,0 +1,7 @@
+export function tableCellStyle(flexGrow: string) {
```

Comment:
Suggest to make this a css class, the properties can be overwritten by making sure the order of the classes in the component is correct.

Also, to not confuse this with other tables, I suggest to name this `useFixedTableCellStyles` or something similar.